k8s/mongodb-backup-agent/container/mongodb_backup_agent_entrypoint.bash

```diff
@@ -12,9 +12,10 @@ if [[ -z "${mms_api_key}" ]]; then
 fi
 
 sed -i '/mmsApiKey/d'  $MONGODB_BACKUP_CONF_FILE
+sed -i '/mothership/d'  $MONGODB_BACKUP_CONF_FILE
 
 echo "mmsApiKey="${mms_api_key} >> $MONGODB_BACKUP_CONF_FILE
+echo "mothership=api-backup.eu-west-1.mongodb.com" >> $MONGODB_BACKUP_CONF_FILE
```

Comment:
The domain cannot be specific to us. It is bad design to have a domain for every customer.
The API keys (which are parameterized) are used to distinguish users of MongoDB Cloud Manager.
Hence, the hardcoded value and not the parameterized value.

Now that you point it out, we might want to parameterize it if some of our users want that in EastUS, rather than EU.
I will file an issue for it.